README.md

```diff
@@ -14,3 +16,47 @@ This lib depends on libgraphqlparser (in submodule in this repository).
 
 You need to build and install this libgraphqlparser binary on your system before using this.
 See [here](https://github.com/graphql/libgraphqlparser) for more details.
+
+## Roadmap - Road to v1
+
+Based on our experience to managing GraphQL API in production for more than one year. We decided to work on a brand new GraphQL engine.
+
+Here are the subjects we are working on:
+
+- [x] Define Open Source guidelines (Contributing, Code of conduct, Issue Template, PR Template).
+- [x] Build communication interface between [libgraphqlparser](https://github.com/graphql/libgraphqlparser) & Tartiflette through [CFFI](https://cffi.readthedocs.io).
+- [x] Build a `Parser` which parse a GraphQL Query and creates a list of Asynchronous Tasks.
+- [x] Build an `Executor` which execute the Async Tasks list created by the `GraphQL Query Parser`.
```

Comment:
which executes.

Use the same names: here you use `GraphQL Query Parser`, above, just `Parser`. Same comment for `Asynchronous Tasks` vs `Async Tasks`

Reply:
Ok